package.json

```diff
@@ -1,18 +1,25 @@
 {
-  "name": "webrtc-signal-http",
-  "version": "1.7.0",
+  "name": "webrtc-signal-http-ts",
+  "version": "1.8.22",
   "description": "opinionated webrtc signal provider using http as a protocol",
-  "main": "lib/index.js",
+  "main": "dist/lib/index.js",
+  "types": "dist/lib/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
   "directories": {
     "lib": "lib",
     "test": "test"
   },
   "bin": {
-    "webrtc-signal-http": "./index.js"
+    "webrtc-signal-http-ts": "./dist/index.js"
```

Comment:
why rename  this?

Reply:
In order to test this, I ran the validation tests on 3D-signal which imports webrtc-signal-http from npm. I created a new package as not to "break" the old one while validating

Reply:
ok - we should not checkin this rename in that case, as we'll use semver in prod to handle that conflict.